### DISCUSSION TITLE
Is there a way to resize heroes/castles list in new interface?

### DISCUSSION THREAD

**Discussion by AAS** [ihhub/fheroes2#4719] [Q&A] [answered]
Hello, is it possible to make heroes list in new interface ("interface: hide" in game options) longer than 2 rows?

![image](https://user-images.githubusercontent.com/179733/143735481-cb1aaa54-d104-4903-a6e9-3a332e356d5f.png)

**Comment by ihhub** [comment, marked as answer]
No. It's fixed. It was designed for small screen resolutions very long time back (before the current team).
